crates/oxc_syntax/src/node.rs

```diff
@@ -1,9 +1,13 @@
 use bitflags::bitflags;
 use nonmax::NonMaxU32;
+use oxc_allocator::CloneIn;
+use oxc_ast_macros::ast;
 use oxc_index::Idx;
+use oxc_span::{cmp::ContentEq, hash::ContentHash};
 #[cfg(feature = "serialize")]
 use serde::{Serialize, Serializer};
 
+#[ast]
```

Comment:
They should be, Previously they were defined via the `define_index_type` macro and we couldn't add them to our schema because we don't support macro expansions - other than inherit ones.

Sorry I wasn't aware of this stack of PRs #4468